TACC/protx
# Task/cooks 73

## Overview: ##

Assign persistent colors to categories assignments.

## Related Jira tickets: ##

* [COOKS-73](https://jira.tacc.utexas.edu/browse/COOKS-73)

## Summary of Changes: ##

Assigned persistent color values to categories of maltreatment, regardless of current selection.

## Testing Steps: ##

1. Build and run project.
2. Add and remove categories from the plots.
3. Ensure the colors assigned to teh categories remain the same regardless of selection or order.

## UI Photos:

## Notes: ##
